lighty: MaintTest.startConfigFileTest FAILURE

```java
[ERROR] startConfigFileTest on startConfigFileTest(io.lighty.controllers.tpce.MaintTest)(io.lighty.controllers.tpce.MaintTest)  Time elapsed: 21.087 s  <<< FAILURE!
java.lang.NoSuchMethodError: 'java.lang.Object org.opendaylight.yangtools.yang.binding.CodeHelpers.requireKeyProp(java.lang.Object, java.lang.String)'
        at io.lighty.controllers.tpce.MaintTest.startConfigFileTest(MaintTest.java:60)
```

`CodeHelpers.requireKeyProp` was added in `yang-binding` ~~**7.0.6**~~ via https://github.com/opendaylight/mdsal/commit/1ee8e403b9c9eb9e8596b5e89cc5ed47676eb71c

~~_**But!**_~~ All current `mdsal` dependencies are set to **7.0.5**

--- a/api/pom.xml
+++ b/api/pom.xml
@@ -12,7 +12,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/common/pom.xml
+++ b/common/pom.xml
@@ -10,7 +10,7 @@
     <parent>
         <groupId>org.opendaylight.mdsal</groupId>
         <artifactId>binding-parent</artifactId>
-        <version>7.0.5</version>
+        <version>7.0.6</version>
         <relativePath />
     </parent>
 
@@ -31,7 +31,7 @@
             <dependency>
                 <groupId>org.opendaylight.mdsal</groupId>
                 <artifactId>mdsal-artifacts</artifactId>
-                <version>7.0.5</version>
+                <version>7.0.6</version>
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>

--- a/inventory/pom.xml
+++ b/inventory/pom.xml
@@ -10,7 +10,7 @@
     <parent>
         <groupId>org.opendaylight.mdsal</groupId>
         <artifactId>binding-parent</artifactId>
-        <version>7.0.5</version>
+        <version>7.0.6</version>
         <relativePath />
     </parent>
 

--- a/networkmodel/pom.xml
+++ b/networkmodel/pom.xml
@@ -27,7 +27,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
       <dependency>
         <groupId>org.opendaylight.mdsal</groupId>
         <artifactId>mdsal-artifacts</artifactId>
-        <version>7.0.5</version>
+        <version>7.0.6</version>
         <scope>import</scope>
         <type>pom</type>
       </dependency>

--- a/olm/pom.xml
+++ b/olm/pom.xml
@@ -13,7 +13,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/ordmodels/common/pom.xml
+++ b/ordmodels/common/pom.xml
@@ -12,7 +12,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/ordmodels/device/pom.xml
+++ b/ordmodels/device/pom.xml
@@ -12,7 +12,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/ordmodels/network/pom.xml
+++ b/ordmodels/network/pom.xml
@@ -12,7 +12,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/ordmodels/pom.xml
+++ b/ordmodels/pom.xml
@@ -12,7 +12,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/ordmodels/service/pom.xml
+++ b/ordmodels/service/pom.xml
@@ -12,7 +12,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/pce/pom.xml
+++ b/pce/pom.xml
@@ -14,7 +14,7 @@
     <parent>
         <groupId>org.opendaylight.mdsal</groupId>
         <artifactId>binding-parent</artifactId>
-        <version>7.0.5</version>
+        <version>7.0.6</version>
         <relativePath />
     </parent>
 

--- a/renderer/pom.xml
+++ b/renderer/pom.xml
@@ -13,7 +13,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/servicehandler/pom.xml
+++ b/servicehandler/pom.xml
@@ -15,7 +15,7 @@ Author: Martial Coulibaly <martial.coulibaly@gfi.com> on behalf of Orange
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/tapi/pom.xml
+++ b/tapi/pom.xml
@@ -15,7 +15,7 @@ Author: Martial Coulibaly <martial.coulibaly@gfi.com> on behalf of Orange
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 
@@ -29,7 +29,7 @@ Author: Martial Coulibaly <martial.coulibaly@gfi.com> on behalf of Orange
       <dependency>
         <groupId>org.opendaylight.mdsal</groupId>
         <artifactId>mdsal-artifacts</artifactId>
-        <version>7.0.5</version>
+        <version>7.0.6</version>
         <scope>import</scope>
         <type>pom</type>
       </dependency>

--- a/tapimodels/pom.xml
+++ b/tapimodels/pom.xml
@@ -12,7 +12,7 @@ and is available at http://www.eclipse.org/legal/epl-v10.html
   <parent>
     <groupId>org.opendaylight.mdsal</groupId>
     <artifactId>binding-parent</artifactId>
-    <version>7.0.5</version>
+    <version>7.0.6</version>
     <relativePath/>
   </parent>
 

--- a/test-common/pom.xml
+++ b/test-common/pom.xml
@@ -12,7 +12,7 @@
     <parent>
         <groupId>org.opendaylight.mdsal</groupId>
         <artifactId>binding-parent</artifactId>
-        <version>7.0.5</version>
+        <version>7.0.6</version>
         <relativePath />
     </parent>
 

